contents/blog/best-open-source-analytics-tools.md

```diff
@@ -11,221 +11,301 @@ featuredImageType: full
+[PostHog](https://posthog.com/) is an all-in-one analytics platform designed to give engineers, product managers and data scientists a complete view of user behavior. It's built on the highly-scalable ClickHouse OLAP database.
 
-Best of all, PostHog can be self-hosted on your existing infrastructure — making it an excellent alternative to tools such as Mixpanel if you’re concerned about privacy. 
+Features include a comprehensive product analytics suite (e.g. trend analysis, dashboards, funnels, path analysis, retention tracking etc.), feature flags for feature control, experimentation (A/B and multivariate testing), session recording, heatmaps and [more](/product).
 
-Of course, we _would_ say PostHog is great, but if you need convincing then you can [get started for free](/pricing) and decide for yourself. You can self-host, but [PostHog Cloud is free up to 1 million events per month](/pricing#cloud) if you'd prefer a fully-managed experience.
+It’s entirely self-serve, with the ability to auto-capture event information out of the box. PostHog can also be self-hosted on your existing infrastructure, making it ideal if you don't want to share data with third parties. It's a good option if you're after an [open-source session recording tool](/blog/best-open-source-session-replay-tools), too. 
 
-### Who is PostHog for?
-PostHog is ideal for engineering, data science, and product teams. As an all-in-one platform, it combines the functionality of traditional product analytics platforms with tools like Feature Flags, Session Recording and A/B testing, making it ideal for teams who want a complete view of their product without the hassle of running multiple separate tools.
+#### Who is PostHog for?
+PostHog is useful for engineering, data science, and product teams. As an all-in-one platform, it can replace multiple tools (e.g. Mixpanel, LaunchDarkly, Hotjar) so you can simplify your tech stack. 
 
-### Strengths
+#### Strengths
 - All-in-one analytics suite
 - Easy-to-use, no SQL required
 - Self-hosting and cloud-hosting available
 - Feature flags for running A/B tests
 - Session recording for gathering insights
 - Event pipelines to integrate with data warehouses
-- Unlimited ability to scale 
+- Built on highly-scalable ClickHouse OLAP database
+
+#### Open source license and monetization
 
-### Open source license
-PostHog Open Source is available under an MIT license. It offers a free version that’s great for start-ups, as well as premium options for large event volumes. Find out more about [self-hosting PostHog](/pricing).
+[PostHog Open Source](https://github.com/PostHog/posthog) is free to use for life and is distributed under an MIT license. The open source version includes the core product analytics, feature flag, and session recording features, but has a one project limit. There's also a [FOSS edition](https://github.com/PostHog/posthog-foss) that removes all proprietary code. 
 
-<ArrayCTA />
```

Comment:
I'd leave this in. Box-outs activate better than text.

Reply:
I dunno. I looked at the numbers and they weren't great. I'd prefer to keep these articles as unsalesy as possible. Might be worth us testing different CTAs

Reply:
Up to you, but I'd still say that low numbers > no numbers 🤷